BREEZE.rst

```diff
@@ -474,61 +453,47 @@ Run pylint checks for all files:
      ./breeze --static-check-all-files pylint
 
 
-The ``license`` check is also run via separate script and separate docker image containing
+The ``license`` check is run via a separate script and a separate docker image containing the 
 Apache RAT verification tool that checks for Apache-compatibility of licences within the codebase.
-It does not take pre-commit parameters as extra args.
+It does not take pre-commit parameters as extra arguments.
 
 .. code-block:: bash
 
      ./breeze --static-check-all-files licenses
 
-Building the documentation
---------------------------
-
-The documentation is build using ``-O``, ``--build-docs`` command:
-
-.. code-block:: bash
-
-     ./breeze --build-docs
-
-Results of the build can be found in ``docs/_build`` folder. Often errors during documentation generation
-come from the docstrings of auto-api generated classes. During the docs building auto-api generated
-files are stored in ``docs/_api`` folder - so that in case of problems with documentation you can
-find where the problems with documentation originated from.
-
-Running tests directly from host
---------------------------------
+Running Tests Directly from the Host
+------------------------------------
 
-If you wish to run tests only and not drop into shell, you can run them by providing
--t, --test-target flag. You can add extra nosetest flags after -- in the commandline.
+If you wish to only run tests and not to drop into shell, you can do this by providing the
+-t, --test-target flag. You can add extra nosetest flags after -- in the command line.
 
 .. code-block:: bash
 
      ./breeze --test-target tests/hooks/test_druid_hook.py -- --logging-level=DEBUG
 
-You can run the whole test suite with special '.' test target:
+You can run the whole test suite with a special '.' test target:
 
 .. code-block:: bash
 
     ./breeze --test-target .
 
-You can also specify individual tests or group of tests:
+You can also specify individual tests or a group of tests:
 
 .. code-block:: bash
 
     ./breeze --test-target tests.core:TestCore
 
-Pulling the latest images
+Pulling the Latest Images
 -------------------------
 
-Sometimes the image on DockerHub is rebuilt from the scratch. This happens for example when there is a
-security update of the python version that all the images are based on.
-In this case it is usually faster to pull latest images rather than rebuild them
-from the scratch.
+Sometimes the image on the Docker Hub needs to be rebuilt from scratch. This is required, for example, when there is a
```

Comment:
not sure my edits are correct here. the original version sounded like the image rebuilt is done automatically.

Reply:
The image in DockerHub is rebuilt automatically when there is new security release of python image. In order to use the new "security patched" images,  it is better to pull the latest version (--force-pull-images) rather than continue with already built images locally. I will update this behaviour slightly to work as follows (maybe it will be easier to explain):
- when new security patched python images are released, the images in DockerHub are rebuilt from the scratch
- when you continue running breeze locally, you will use the already downloaded images (based old python version).
- if you want to update to latest security-patched images you need to run breeze with "--force-pull-images" flag.